lib/rules/no-restricted-html-elements.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @author Doug Wade <douglas.b.wade@gmail.com>
+ */
+
+'use strict'
+
+const utils = require('../utils')
+
+module.exports = {
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description: 'prevent usage of restricted html elements',
```

Comment:
To be consistent with other `no-restricted-*` rules:

```suggestion
      description: 'disallow specific HTML elements',
```

Please also run `npm run update` afterwards to update the docs, too.